src/hotspot/share/oops/method.cpp

```diff
@@ -226,6 +226,12 @@ void Method::print_external_name(outputStream *os, Klass* klass, Symbol* method_
 }
 
 int Method::fast_exception_handler_bci_for(const methodHandle& mh, Klass* ex_klass, int throw_bci, TRAPS) {
+  bool do_logging = log_is_enabled(Info, exceptions);
+  if (do_logging) {
+    ResourceMark rm(THREAD);
+    log_info(exceptions)("Looking for catch handler for exception of type \"%s\" in method \"%s\"",
```

Comment:
I think these should be log_debug, since this is useful for debugging but many people outside of hotspot engineers use -Xlog:exceptions.

Reply:
If the exception is not immediately caught, will this print a line for every method on the call stack that has a catch block? If so, that may be too verbose.